packages/studio-ui/src/web/views/FlowBuilder/nodeProps/StandardNode.tsx

```diff
@@ -0,0 +1,140 @@
+import { Tag } from '@blueprintjs/core'
+import { lang, Tabs, toast } from 'botpress/shared'
+import { FlowView, NodeView } from 'common/typings'
+import React, { useState } from 'react'
+
+import { sanitizeName } from '~/util'
+
+import EditableInput from '../common/EditableInput'
+
+import ActionSection from './ActionSection'
+import style from './style.scss'
+import TransitionSection from './TransitionSection'
+
+type TabId = 'on-enter' | 'on-receive' | 'transitions'
+
+interface Tab {
+  id: TabId
+  disabled?: boolean
+  className?: string
+  title: string | JSX.Element
+}
+
+interface Props {
+  flow: FlowView
+  node: NodeView
+  subflows: string[]
+  readOnly: boolean
+  isLastNode: boolean
+  updateNode: (node: Partial<NodeView>) => void
+  copyFlowNodeElement: Function
+  pasteFlowNodeElement: Function
+  buffer: any
```

Comment:
Can you try to improve those typings?

Reply:
I have no idea what they are at the moment but they are used everywhere, I'll try to fix